### DISCUSSION TITLE
Graphing Window features like Vector CANoe graphics tool

### DISCUSSION THREAD

**Discussion by Yash7764** [collin80/SavvyCAN#809] [Ideas]
hii...i am having a suggestion, since i am new to this, i am trying to add cursor option in graphing window just like other CAN analysis tools are having, it will help more in analysis, also the value axis is only one and when we are analysing signals with very high value, small value signals look like zero, i am thinking of adding each value axis for each variable/signal which can be zoomed separately to analyse high value signal with low value signal just like other CAN analysis tools are having this feature, if anyone has already done that, kindly share the code

**Comment by collin80** [comment]
You're right, it's problematic when you need to graph signals of very different magnitudes. What I have been doing is scaling one or more of the graphs so that they are closer in magnitude to the others. This, of course, messes up the actual values but if you scale by 10x or 100x then it's just moving the decimal point. As for the cursor, this does currently exist. Double click on a graph and it should give you the X and Y value nearest the point where you clicked.
